src/App.js

```diff
@@ -3,6 +3,14 @@ import './App.css';
 
 import todos from './api/todos';
 import users from './api/users';
+import { TodoList } from './components/TodoList/TodoList';
+
+const preparedTodos = todos.map(todo => (
+  {
+    ...todo,
+    user: users.find(user => user.id === todo.userId),
+  }
+));
```

Comment:
Дімі не сподобається роби ось так 
```suggestion
const preparedTodos = todos.map(todo => ({
   ...todo,
   user: users.find(user => user.id === todo.userId),
 }));
```